recruiter/forms.py

```diff
@@ -0,0 +1,29 @@
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth.models import User
+from .models import Recruiter, Company
+
+
+class RecuiterRegistrationForm(UserCreationForm):
```

Comment:
Does this still need to inherit from UserCreationForm? Can it me a simpler ModelForm without all the custom overrides now that you made separate screens?

Reply:
I use the UserCreationForm inherit when I sent the request.POST argument in the views.py file. that's the way I saw how to do it (I can send the youtube link if it's relevant), maybe you know a better or different way to do it?

Reply:
Are you using a single form to both create a user account and register it as a recruiter or two different forms, one for creating a user and the other to set it as a recruiter? 

If its two different forms, then there is no reason for the recruiter registration for to inherit from the user creation form.

Reply:
I'm using a single form for both, create a user account and register it as a recruiter, I'm also testing this form in the tests.